src/openms/include/OpenMS/CHEMISTRY/IsotopeDistribution.h

```diff
@@ -155,6 +155,78 @@ namespace OpenMS
     */
     void estimateFromWeightAndComp(double average_weight, double C, double H, double N, double O, double S, double P);
 
+
+    /**
+        @brief Estimate peptide fragment IsotopeDistribution from the precursor's average weight,
+        fragment's average weight, and a list of isolated precursor isotopes.
```

Comment:
It was to keep it flexible. It's technically possible to perform a multiplexed scan where the mass spec performs two separate isolations of different m/z and then scans them together. So maybe someone would want to isolate isotopes M0-1 and M3-4. I've never seen that done though so maybe using a list is more trouble than it's worth. I can certainly switch it to a range and if one day someone needs a discontinuous version they could write their own function that calls the ranged version with various intervals and then merges the results.